mhlib/app.go

```diff
@@ -191,6 +197,9 @@ func (a *App) apply(configFile string, simulate bool) (*[]interface{}, error) {
 	// "performs pods restart for the resource if applicable"
 	cmd = append(cmd, []interface{}{"--recreate-pods"}...)
 
+	// "namespace to install the release into. (Default: 'default')"
+	cmd = append(cmd, []interface{}{"--namespace", a.Namespace}...)
```

Comment:
continuing the nit.. would you consider the `--namespace` to be optional and check for it rather here then above?
```go
if a.Namespace != "" {
    cmd = append(cmd, []interface{}{"--namespace", a.Namespace}...)
}
```

Reply:
I think it can't break anything at the moment, I will submit a PR with this change later to drive the discussion further.